pkg/test/resourcefixture/testdata/basic/dataform/v1alpha1/dataformrepository/_generated_object_dataformrepository.golden.yaml

```diff
@@ -16,12 +15,11 @@ spec:
   projectRef:
     external: ${projectId}
   region: us-west2
-  resourceID: dataformrepository-${uniqueId}
```

Comment:
cc @yuwenma ... I think we should decide whether we are going to write-back spec.resourceID in the long run.  (My preference would be no, but I think we need to start populating status.externalRef)

Reply:
"write-back spec.resourceID" --> No, we will never do that! Using externalRef has been record in the internal manual. I'll pull out a doc to talk about the rational, decision and plans.